tests/app/test_mqtt_client.py

```diff
@@ -22,7 +21,3 @@ def test_run(self):
 
         # Verify that connect was called with the correct parameters
         self.mock_mqtt_client.connect.assert_called_with("localhost", 1883)
-
-
-if __name__ == "__main__":
```

Comment:
when is this not needed? I am not fluent in Python..

Reply:
It's not needed because we use pytest to execute the module e.g. `pytest tests/app/test_mqtt_client.py`, rather than with python itself, so we never end up in `main` scope